### PR TITLE
Listen to HTTP requests

### PR DESCRIPTION
**Pull request by kchulj**
Faucet listens to HTTP requests so that the endpoints can be called.

Solves #27

### CODE REVIEW

**Comment by kchulj** [comment]
Ping for review.

File: source/faucet/main.d [hunk 1]
```diff
@@ -48,6 +50,12 @@ private struct Config
 
     /// Between how many addresses we split a transaction by
     static immutable count = 15;
+
+    /// Bind address
+    public string address = "0.0.0.0";
+
+    /// Bind port
+    public ushort port = 0xACE;
```
Comment (linked0) [inline source/faucet/main.d:58]: I think we usually use the decimal number for a port number, not hexadecimal. Don't we?

Comment (kchulj) [reply source/faucet/main.d:58]: Correct. But I saw `0xBOA` in agora :D

**Review by linked0 — APPROVED** [review]
LGTM

File: source/faucet/main.d [hunk 2]
```diff
@@ -48,6 +50,12 @@ private struct Config
 
     /// Between how many addresses we split a transaction by
     static immutable count = 15;
+
+    /// Bind address
+    public string address = "0.0.0.0";
+
+    /// Bind port
```
Comment (Geod24) [inline source/faucet/main.d:57]:
```suggestion
    /// Bind port (default: 2766)
```

File: source/faucet/main.d [hunk 3]
```diff
@@ -378,6 +386,15 @@ int main (string[] args)
     logInfo("The address of node is %s", args[1]);
     auto faucet = new Faucet(Config.init, args[1]);
     faucet.setup(faucet.config.count);
+
+    setLogLevel(LogLevel.diagnostic);
```
Comment (Geod24) [inline source/faucet/main.d:390]: That's a bit verbose

**Review by Geod24 — APPROVED** [review]

**Comment by kchulj** [comment]
Updated and green.

**Pull request merged**